hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/testutils/HiveTestUtil.java

```diff
@@ -120,48 +120,45 @@ public static void setUp() throws IOException, InterruptedException {
     clear();
   }
 
-  public static void clear() throws IOException {
+  public static void clear() throws Exception {
     fileSystem.delete(new Path(hiveSyncConfig.basePath), true);
     HoodieTableMetaClient.withPropertyBuilder()
-      .setTableType(HoodieTableType.COPY_ON_WRITE)
-      .setTableName(hiveSyncConfig.tableName)
-      .setPayloadClass(HoodieAvroPayload.class)
-      .initTable(configuration, hiveSyncConfig.basePath);
+        .setTableType(HoodieTableType.COPY_ON_WRITE)
```

Comment:
Please revert formatting changes. When we pull in this PR, it will cause more merge conflicts in the future for lines that actually did not change.

Reply:
This is the only place where I have left it as is because it was according to the checkstyle spec.